docs/InputField.md

```diff
@@ -1,23 +1,36 @@
-# InputField
+#FormField alias FormField
```

Comment:
Markdown syntax requires a space after the `#`. Also should this say `FormField` then list the alias below?

Reply:
Webstorm's `md` preview feature ignores that kind of stuff - making the fix.